### ISSUE TITLE
Detail question of regularization in Roboflow-VL Few-Shot Object Detection Challenge 2025

### ISSUE THREAD

**Issue by SangbumChoi** [anishmadan23/foundational_fsod#7]
@neeharperi Hi I have several question about regularization in this challenge.

> Environment for model development:
Pretraining: Models are allowed to pre-train on any existing datasets.
Fine-Tuning: Models can fine-tune on 10 shots from each of RF20-VL's datasets
Evaluation: Models are evaluated on RF20-VL's test set. Each dataset is evaluated independently.

This is the quote from the https://github.com/roboflow/rf100-vl?tab=readme-ov-file#benchmarking-protocols

- For the pretraining dataset, is it allowed to use any dataset except for the RF20-VL dataset?
- For fine-tuning stage, do we have specific 10-shot examples?
- (If above is not true) Can user extract any 10-shot from train/val **(except test)** from the RF20-VL?
- For the word 'fine-tuning', does it mean litterally update weight parameter from the 10 shot example (I understand 10-shot means 10 objects which is irrelevant to the number of images)? or does it mean just extracting prompt example to predict the evaluation dataset?
- For the evaluation protocol can we assume that it is similar to COCOEval?

Thanks for preparing this competition 👍🏼

**Comment by neeharperi** [comment]
Thanks for your interest in our challenge!

- You can use any dataset except RF20-VL for pre-training. 
- You must use the specific 10-shots defined in the RF20-VL-FSOD train-set. You're correct that each class has 10 bounding boxes, which is irrelevant to the number of images.
- Fine-tuning can be any learning algorithm (e.g. in-context learning, gradient-based fine-tuning, prompt tuning, etc.) that can adapt you rpre-trained model to the target dataset. 
- We use the same evaluation protocol as COCO

Please let me know if you have any addition questions.
